### PR TITLE
Upgrade polkadot to v0.9.40

### PR DESCRIPTION
**Pull request by devdanco**

### CODE REVIEW

File: bencher/src/bench_ext.rs [hunk 1]
```diff
@@ -9,7 +8,7 @@ use super::tracker::BenchTracker;
 
 pub struct BenchExt<'a, H, B>
 where
-	H: Hasher,
+	H: sp_core::Hasher,
```
Comment (xlc) [inline bencher/src/bench_ext.rs:11]: just replace `use hash_db::Hasher;` with `use sp_core::Hasher;`

**Comment by devdanco** [comment]
@xlc, I noticed there are failing warnings in the cargo clippy -- -D warnings command inside the CI build. Specifically, I am receiving the following warning: 
`"use of deprecated associated function frame_support::dispatch::Weight::from_ref_time: Will be removed soon; use from_parts instead."`
 Since these files are autogenerated, should I clone the Acala node, adjust it to my branch, and run the benchmarks? Thank you.

**Comment by xlc** [comment]
let’s just manually edit those files for now and we can update the templates in Acala and regenerate the weight files later

**Comment by xlc** [comment]
You can ignore that. It is not a blocking issue and we will fix it later.

**Review by zjb0807 — APPROVED** [review]
LGTM

**Comment by xlc** [comment]
Thanks for the contribution!

Release branch [polkadot-v0.9.40](https://github.com/open-web3-stack/open-runtime-module-library/tree/polkadot-v0.9.40) is created.

**Pull request merged**